scripts/mocha-coverage.js

```diff
@@ -13,7 +13,7 @@ const MOCHA_BIN = path.join(path.dirname(__dirname), 'node_modules', '.bin', 'mo
 const NYC_BIN = path.join(path.dirname(__dirname), 'node_modules', '.bin', 'nyc')
 
 let bin = NYC_BIN
-let argv = ['--cache', MOCHA_BIN]
+let argv = ['--cache', '--no-clean', MOCHA_BIN]
```

Comment:
What does this do? (purely for my own curiosity, again)

Reply:
The `--no-clean` does not remove coverage files from previous runs. Since I split the test runs, it generates the coverage data as if it was run in one operation.